go/test/endtoend/vreplication/wrappers_test.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vreplication
+
+import (
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"vitess.io/vitess/go/vt/log"
+)
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
```

Comment:
This is setting the stage for further refactoring of the e2e testing. Only the functionality required for the FK test is currently implemented. We will implement the others as we port over other tests.